File: Python/Array/Find Numbers with Even Number of Digits.py
class Solution:
    def check(self, num: int) -> bool:
        cnt = 0
        while num != 0:
            cnt += 1
            num //= 10
        if cnt % 2 == 0:
            return True
        return False

    def findNumbers(self, nums: List[int]) -> int:
        cnt = 0
        for num in nums:
            if self.check(num):
                cnt += 1
        return cnt

File: Java/Array/Find Numbers with Even Number of Digits.java
class Solution {
    boolean check(int num) {
        int cnt = 0;
        while (num != 0) {
            cnt++;
            num /= 10;
        }
        if (cnt % 2 == 0)
            return true;
        return false;
    }

    public int findNumbers(int[] nums) {
        int cnt = 0;
        for (int num : nums) {
            if (check(num))
                cnt++;
        }
        return cnt;
    }
}
